source/monitoring/logging.rst

```diff
@@ -0,0 +1,94 @@
+.. _logging:
+
+Logging
+=======
+
+This page provides an overview of the locations Open OnDemand writes logs and useful session data and 
+provides context around these locations.
+
+In general administrators will want to note whether they need to see:
+
+- System type logs for authentication to the :ref:`web-node <glossary>` itself or OOD's own configuration.
+- Session data for things like connecting to interactive apps (Jupyter, Rstudio, Codeserver, etc.) from 
+  the user's PUN, which is already running on the web-node.
+
+System Logs
+-----------
+These logs will provide information around authentication and internal Open OnDemand errors. They are very useful 
+when initially installing and configuring Open OnDemand.
+
+**Apache Logs**
+
+For logs relating to issues about: 
+
+ - Authenticating to the web-node
+ - Configuring OnDemand 
+
+Look in either of:
+
+:file:`/var/log/httpd/<hostname>_error.log`
+
+- OnDemand will log to this location for its own configuration errors.
+
+:file:`/var/log/httpd/<hostname>_access.log`
+
+- Where OnDemand will log succsseful logins.
+
+.. warning::
+
+    There are no entries for failed logins.
+
+**NGINX Logs**
+
+For logs relating to issues about:
+
+ - Users trying to connect to a :ref:`login-node <glossary>` from their PUN check:
+
+:file:`/var/log/ondemand-nginx/<user>`
+
+.. note::
+
+    These logs by default have ``root:root`` ownership. It may be beneficial 
+    to ``chown`` an appropriate staff group on these files for troubleshooting down the road.
+
+Session Data
+------------
+Though not logs per-se, these files will provide information around connections between the user's PUN and a 
+:ref:`login-node<glossary>`, such as an interactive app like Jupyter.
```

Comment:
So jobs run on compute nodes, not login nodes so I'd just remove that bit.

They also _are_ logs becuase that's where we tell the scheduler to put the stdout and stderr and apps typically log other info there.

Lastly - they're `information around connections` so much as they are `the working directory of the job as well as the location of stdout and stderr output files (typically just output.log)`.

Reply:
Updated to reflect the correct node type and also tried to capture the connections and location information as well. Let me know what you think.